calc-sintaxis.y

```diff
@@ -594,9 +600,14 @@ char * get_string_representation(ASTNode * node) {
 */
 void print_tree_formatted_by_level(ASTNode *root, int level) {
   if (root != NULL) {
+    TypeNode node_type = root -> node_type;
+    if (!(node_type == _if ||node_type == _while || node_type == _if_body))
```

Comment:
We Should delete all printf calls used to debug before merge.

Reply:
Thats the function that prints the AST, I dont see any debug printf :)

Reply:
You're right, my mistake. Sorry